linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/execute/DefaultEntranceExecutor.scala

```diff
@@ -343,4 +324,108 @@ class DefaultEntranceExecutor(id: Long)
     }
   }
 
+  def requestToComputationJobReq(
+      entranceExecuteRequest: EntranceExecuteRequest
+  ): ComputationJobReq = {
+    val jobReqBuilder = ComputationJobReq.newBuilder()
+    jobReqBuilder.setId(entranceExecuteRequest.jobId())
+    jobReqBuilder.setSubmitUser(entranceExecuteRequest.submitUser())
+    jobReqBuilder.setExecuteUser(entranceExecuteRequest.executeUser())
+    val codeTypeLabel: Label[_] = LabelUtil.getCodeTypeLabel(entranceExecuteRequest.getLabels)
+    if (null == codeTypeLabel) {
+      throw new EntranceErrorException(
+        EntranceErrorCode.EXECUTE_REQUEST_INVALID.getErrCode,
+        s"code Type Label is needed"
+      )
+    }
+    val codes = new util.ArrayList[String]()
+    codes.add(entranceExecuteRequest.code())
+    val codeLogicalUnit =
+      new CodeLogicalUnit(codes, codeTypeLabel.asInstanceOf[CodeLanguageLabel])
+    jobReqBuilder.setCodeLogicalUnit(codeLogicalUnit)
+    jobReqBuilder.setLabels(entranceExecuteRequest.getLabels)
+    jobReqBuilder.setExecuteUser(entranceExecuteRequest.executeUser())
+    jobReqBuilder.setParams(entranceExecuteRequest.properties())
+    jobReqBuilder.build().asInstanceOf[ComputationJobReq]
+  }
+
+  def callSimpleExecute(request: ExecuteRequest): ExecuteResponse = {
```

Comment:
This method lacks log, process, and result response handlers, see EngineExecuteAsyncReturn